frontend/src/stories/ToggleSwitch.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import type { Meta, StoryObj } from '@storybook/react-webpack5';
+import { useState } from 'react';
+import { ReactComponent as CheckMarkIcon } from '../@assets/icons/check.svg';
+import { ReactComponent as XMarkIcon } from '../@assets/icons/close.svg';
+import ToggleSwitch from '../components/@common/toggle/ToggleSwitch';
+import { theme } from '../styles/theme';
+
+const meta: Meta<typeof ToggleSwitch> = {
+  title: 'Components/ToggleSwitch',
+  component: ToggleSwitch,
```

Comment:
꼭 반영할 필요는 없지만 스토리북 팁을 공유해보자면,
parameters로 layout 설정을 추가하면 스토리북의 중앙에서 볼 수 있어요!

```suggestion
  component: ToggleSwitch,
  parameters: {
    layout: 'centered',
  },
```

<img width="612" height="502" alt="CleanShot 2025-09-19 at 16 51 23@2x" src="https://github.com/user-attachments/assets/b9ec317b-6a94-4e08-b481-6157d1457d8b" />

Reply:
https://github.com/woowacourse-teams/2025-PhotoGather/pull/612/commits/cfba2ed341b602586f99d2c45a5297d5d1e14c18 전혀 몰랐던 설정이네요! layout 설정 완료했습니다~